tensorflow/compiler/xla/tests/matrix_ops_simple_test.cc

```diff
@@ -234,16 +237,208 @@ class MatOpsDotAddTest
                            {lhs_handle.get(), rhs_handle.get()},
                            ErrorSpec(1e-6));
   }
+
+  template <typename T>
+  void iota_int_init_value(std::vector<T>& values, int init_value) {
+    absl::c_for_each(values,
+                     [&](T& value) { value = static_cast<T>(init_value++); });
+  }
+
+  template <typename T>
+  XlaOp generate_dot(XlaBuilder& builder, Array2D<T>& lhs, Array2D<T>& rhs,
+                     bool transpose) {
+    auto prim_type = primitive_util::NativeToPrimitiveType<T>();
+    Shape lhs_shape =
+        ShapeUtil::MakeShape(prim_type, {lhs.height(), lhs.width()});
+    Shape rhs_shape =
+        ShapeUtil::MakeShape(prim_type, {rhs.height(), rhs.width()});
+
+    auto lhs_arg = Parameter(&builder, 0, lhs_shape, "lhs");
+    auto lhs_mat_arg = lhs_arg;
+    if (transpose) {
+      lhs_mat_arg = Transpose(lhs_mat_arg, {1, 0});
+    }
+    auto rhs_arg = Parameter(&builder, 1, rhs_shape, "rhs");
+    auto dot = Dot(lhs_mat_arg, rhs_arg);
+    return dot;
+  }
+
+  template <typename T>
+  XlaOp generate_dot_add(XlaBuilder& builder, Array2D<T>& lhs, Array2D<T>& rhs,
+                         bool transpose, int n) {
+    auto dot = generate_dot(builder, lhs, rhs, transpose);
+    auto prim_type = primitive_util::NativeToPrimitiveType<T>();
+    Shape bias_shape = ShapeUtil::MakeShape(prim_type, {n});
+    auto bias_arg = Parameter(&builder, 2, bias_shape, "bias");
+    auto bias_bcast = BroadcastInDim(bias_arg, {n, n}, {1});
+    auto add_result = Add(dot, bias_bcast);
+    return add_result;
+  }
+
+  template <typename T>
+  void TestImplBiasAddEpilogueFusion() {
+    bool row_major = std::get<0>(GetParam());
+    bool transpose = std::get<2>(GetParam());
+    bool use_cublaslt = std::get<3>(GetParam());
+    execution_options_.mutable_debug_options()->set_xla_gpu_enable_cublaslt(
+        use_cublaslt);
+    Array2D<T> lhs({{1.0f, 2.0f}, {3.0f, 4.0f}});
+    Array2D<T> rhs({{10.0f, 11.0f}, {12.0f, 13.0f}});
+    auto minor_to_major = [](bool row_major) -> std::vector<int64_t> {
+      return {row_major ? 1 : 0, row_major ? 0 : 1};
+    };
+
+    XlaBuilder builder(TestName());
+    auto result = generate_dot_add(builder, lhs, rhs, transpose, 2);
```

Comment:
result is unused

Reply:
The alternative would be to ignore the return value. I don't think it's preferable.

Reply:
@SandSnip3r see also [this existing test](https://github.com/tensorflow/tensorflow/blob/5539d3a1c7a853783245d1cb71c61e8d931d232c/tensorflow/compiler/xla/tests/matrix_ops_simple_test.cc#L218). Do you want to remove the variable?

Reply:
Unused variables are treated as a compiler error.

I have made this change internally and also another change. I am trying to get the internal build to pass. Don't worry about making any changes for now. If I run into any other issues, I'll ping you here.